src/project/SidebarView.js

```diff
@@ -90,6 +89,8 @@ define(function (require, exports, module) {
      * Show the sidebar.
      */
     function show() {
+	$sidebar.width(MinimumSidebarWidth); 
```

Comment:
Okay, I completely agree the if statement is needed. If the file tray width is more than 200px and  the user minimizes the file tray and then expands it. it goes back to the width the user had set before